static/sass/components/_intro.scss

```diff
@@ -0,0 +1,64 @@
+// Intro Blocks
+// ============
+// Two column (on wide screens) intro and aside
+// Can be seen on case studies
+
+// wrapping container for intro
+.intro-block {
+  @include grid;
+  display: flex;
+  flex-direction: column;
+
+  @include above('small-page') {
+    @include center('medium-page');
+
+    @supports not (display: grid) {
+      flex-direction: row;
+      justify-content: flex-end;
+    }
+  }
+}
+
+// intro inner wrapper text block and task info
+// using this as a container to use flexbox inside and grid outside
+.intro {
+  @supports (display: grid) {
+    grid-column: col1-start / -1;
+  }
+
+  @include above('small-page') {
+    display: flex;
+  }
+
+  @include above('large-page') {
+    @supports not (display: grid) {
+      max-width: size('wide-column');
+    }
+
+    grid-column: col2-start / -1;
+  }
+
+  p {
+    font-style: italic;
+
+    @include above('small-page') {
+      flex: 1 0 0;
+      padding-right: size('gutter');
+    }
+  }
+}
+
+// intro side panel
+.project-tasks {
```

Comment:
I agree. I think we need to add some spacing and maybe design or font size shift ... something